src/cli-storage.ts

```diff
@@ -0,0 +1,19 @@
+import fs from "node:fs";
+import os from "node:os";
+
+const __confDir = os.homedir() + "/.baseplate";
+if (!fs.existsSync(__confDir)) {
+  fs.mkdirSync(__confDir);
+}
+
+export async function storageProvider() {
+  // @ts-ignore
```

Comment:
What typescript error is being suppressed here? Why isn't conf being imported as a static import?

Reply:
It xould be done as part of #29